src/export/InstanceExporter.ts

```diff
@@ -264,6 +264,24 @@ export class InstanceExporter implements Fishable {
       fshDefinition
     );
     cleanResource(instanceDef);
+
+    // check for another instance of the same type with the same id
+    // see https://www.hl7.org/fhir/resource.html#id
+    // instanceDef has already been added to the package, so it's fine if it matches itself
+    if (
+      this.pkg.instances.some(
+        instance =>
+          instanceDef.resourceType === instance.resourceType &&
+          instanceDef.id === instance.id &&
+          instanceDef !== instance
```

Comment:
I think this could become a problem with #464. In that PR, we are adding support for instances of Extensions and Types (like a Quantity or CodeableConcept). But if the user is making an instance of a Quantity, I did not want to force them to put an id on that instance, since they likely don't want an id by default, and that instance is not actually being exported to JSON. So in that case I think two inline instances of Quantity which both would not have a `resourceType` and would likely not have an `id` (unless the user chose to set one for some reason) could fall into this case, both those fields on both instances would be null.

So I think here you want to exclude Inline instances, since they are not standing alone as resources.